realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.java

```diff
@@ -478,11 +478,11 @@ private void emitCopyOrUpdateMethod(JavaWriter writer) throws IOException {
                 className, // Return type
                 "copyOrUpdate", // Method name
                 EnumSet.of(Modifier.PUBLIC, Modifier.STATIC), // Modifiers
-                "Realm", "realm", className, "object", "boolean", "update" // Argument type & argument name
+                "Realm", "realm", className, "object", "boolean", "update", "Map<RealmObject,RealmObject>", "cache" // Argument type & argument name
```

Comment:
No, that doesn't work.

I have been thinking about implementing a interface as well. All our static methods could move into it as well, which could reduce all our reflection calls in Realm atm. But that should probably happen as part of a another PR?